numpy/linalg/lapack_lite/make_lite.py

```diff
@@ -10,6 +10,7 @@
  * f2c
  * patch
 
+WARNING: This a Python 2 script. Read README.rst for rationale.
```

Comment:
I have moved the warning at the top of the file right under the shebang, in a comment instead of a docstring.

I find this more visible. Which do you prefer?

Reply:
Yes, this seems sensible to me.